mathics/core/definitions.py

```diff
@@ -107,7 +107,7 @@ def __init__(
             "Global`",
         )
 
-        from mathics.builtin.pymathics import (
+        from mathics.eval.pymathics import (
```

Comment:
It was my understanding that the idea behind `eval` was to transition to a model where we could run a sequence of instructions where each instruction corresponds to an eval function right now. 

I am not opposed to moving around the functions that load modules or separating pymathics things, but I don't get why the natural place for this is in `mathics.eval`

Reply:
I put it here because is what if follows from evaluating an expression like ``LoadModule[module]``. In any case, this was in ``mathics.builtin.pymathics`` and it does not belong there. Maybe it belongs to ``mathics.core.pymathics``.

Reply:
> I put it here because is what if follows from evaluating an expression like `LoadModule[module]`. In any case, this was in `mathics.builtin.pymathics` and it does not belong there. Maybe it belongs to `mathics.core.pymathics`.

Ok. Sure `mathics.core` or somewhere else is fine. Probably some of the things in as yet not finished `mathics.builtin.system_init` belong in core as well.

Reply:
Done in [5caa0c3](https://github.com/Mathics3/mathics-core/pull/630/commits/5caa0c383a8dc11197bb350723d387f26c80b0dd)